portfolio/src/main/webapp/index.html

```diff
@@ -3,55 +3,172 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <!-- Scripts -->
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
     <script src="script.js"></script>
   </head>
   <body>
-    <nav>
-        <input type="checkbox" id="check">
-        <label for="check" class="checkbtn">
-            <i class="fas fa-bars"></i>
-        </label>
-        <label class="logo">Brandocean ⚡️</label>
-        <ul>
-            <li><a class="active" href="#">Home</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Projects</a></li>
-            <li><a href="#">Experience</a></li>
-            <li><a href="#">Contact</a></li>
-            <li><a href="#">Feedback</a></li>
-            <li><a href="Experiments.html">Experiments</a></li>
-        </ul>
-    </nav>
-    <div id="content">
-      <h1>Brandon's Portfolio</h1>
-      <img class="center" src="/images/Brandocean.jpg" width="300" height="300"/>
-      <h2>Who am I?</h2>
-      <p>Hi! I'm Brandon, I'm from Morelos, Mexico. 
-          I joined to SPS program to improve my skills, meet new people and have fun because I love my bachelor degree. 
-          I like watching anime (my favourite Danmachi) and have fun with my friends.</p>
-      <h2>What makes me me?</h2>
-      <p>I am a noble person, persistent, friendly, methodical, enterprising and with leadership skills. 
-          Always having in my mind the phrase “nothing is impossible, just a little challenging”</p>
-      <h2>My projects</h2>
-      <ul>
-        <li><b>Plant monitoring with IoT:</b> October-November 2021. Web Service
-            that shows the variables of a plant-like temperature, sunlight, humidity
-            in real-time</li>
-        <li><b>Thermal Investment Challenge:</b> September-October 2020. Data
-            analysis on the phenomenon of thermal inversion with 3D modeling in
-            MATLAB.</li>
-        <li><b>Challenge Innovation in product design in the glass industry:</b>
-            February-March 2021. Construction of an innovative design of glass
-            including a circuit of LED lights.</li>
-      </ul>
-      <h2>Contact me</h2>
-      <ul>
-          <li><a href="https://www.linkedin.com/in/brandon-rivera-239915179/">Linkedin</a></li>
-          <li><a href="https://github.com/Brandocean">GitHub</a></li>
-      </ul>
+    <!-- Hero section -->
+    <div class="hero">
+		<nav>
+			<h2 class="logo">Portfo<span>lio</span></h2>
+			<ul>
+				<li><a href="index.html">Home</a></li>
+				<li><a href="index.html#about">About Us</a></li>
+				<li><a href="index.html#service">Projects</a></li>
+				<li><a href="index.html#">Experience</a></li>
+				<li><a href="index.html#contact-me">Contact Us</a></li>
```

Comment:
Since this is already a part of `index.html` the `href` can simply have the `#id`